media/views/featureSelection/featureSelection.css

```diff
@@ -17,3 +17,5 @@ body {
 button {
   margin-block-start: 1em;
 }
+
+/*# sourceMappingURL=featureSelection.css.map */
```

Comment:
Minor: Do you need that commented in the code?

Reply:
Actually nope, I've removed source maps as of no need atm. Good catch.